src/io/flutter/actions/OpenInXcodeAction.java

```diff
@@ -104,9 +111,25 @@ public void processTerminated(@NotNull ProcessEvent event) {
     }
   }
 
-  private static boolean hasBeenBuilt(@NotNull PubRoot pubRoot) {
-    final VirtualFile buildDir = pubRoot.getRoot().findChild("build");
-    return buildDir != null && buildDir.isDirectory() && buildDir.findChild("ios") != null;
+  private static boolean hasBeenBuilt(@NotNull PubRoot pubRoot, @NotNull FlutterSdkVersion version) {
+    if (version.isXcodeConfigOnlySupported()) {
+      // Derived from packages/flutter_tools/test/integration.shard/build_ios_config_only_test.dart
+      final VirtualFile ios = pubRoot.getRoot().findChild("ios");
+      if (ios == null || !ios.isDirectory()) return false;
+      final VirtualFile flutter = ios.findChild("Flutter");
+      if (flutter == null || !flutter.isDirectory()) return false;
+      final VirtualFile gen = flutter.findChild("Generated.xcconfig");
+      if (gen == null || gen.isDirectory()) return false;
+      try {
+        final String contents = new String(gen.contentsToByteArray());
+        return contents.contains("DART_OBFUSCATION=true");
```

Comment:
This is only true in `build_ios_config_only_test.dart` because `--obfuscate` was passed into the integration test.  
https://github.com/flutter/flutter/blob/99472cf072f154a6698c9795918ce480d49ceb57/packages/flutter_tools/test/integration.shard/build_ios_config_only_test.dart#L31
You shouldn't rely on this value being present or set to `true`.